Terminal.Gui/ConsoleDrivers/V2/Logging.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Diagnostics.Metrics;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+
+namespace Terminal.Gui;
+
+/// <summary>
+///     Singleton logging instance class. Do not use console loggers
+///     with this class as it will interfere with Terminal.Gui
+///     screen output (i.e. use a file logger).
+/// </summary>
+/// <remarks>
+///     Also contains the
+///     <see cref="Meter"/> instance that should be used for internal metrics
+///     (iteration timing etc).
+/// </remarks>
+public static class Logging
```

Comment:
Love this concept. 

Does it have to be restricted to drivers though?

Reply:
Certainly this can be used later on for other bits of the core code, it is not tied to drivers specifically.

Reply:
I only put it in this Drivers folder because its where I was creating everything else, can move to root or another folder if you like.